.github/workflows/tests-datapath-verifier.yaml

```diff
@@ -87,9 +87,6 @@ jobs:
       matrix:
         include:
           # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
-          - kernel: '5.4-20250428.152446'
-            ci-kernel: '54'
-          # renovate: datasource=docker depName=quay.io/lvh-images/complexity-test
           - kernel: 'rhel8.6-20250428.152446'
             ci-kernel: '54'
```

Comment:
Would it make sense to bump the `ci-kernel` for RHEL8.6 to `510` given that is our new minimum?

Reply:
I suppose so :). Let's take that as a "CI consolidation" follow-up :+1: